### ISSUE TITLE
Broken link for "MXNet R Reference Manual"?

### ISSUE THREAD

**Issue by dslate** [apache/incubator-mxnet#8344]
On the MXNet page for the R API (http://mxnet.incubator.apache.org/api/r/index.html) the link to the "MXNet R Reference Manual" seems to point to a document called "mxnet-test.pdf", whose contents are titled "MXNet Documentation Release 0.0.8" and seem to be for the Julia language, not R.

Is this link incorrect, or am I just confused?  I am looking for the MXNet R API.

Thanks.

**Comment by xup6fup** [comment]
I have written some examples using MxNetR for implementing regression  recently, and I will add more examples later.
https://github.com/xup6fup/MxNetR-examples

**Comment by liori** [comment]
Duplicate of #7483?

**Comment by lanking520** [comment]
@chetkhatri @piiswrong 
Hi Eric and Chetan,
do we have any way to recover the old R manual?
Reference: https://github.com/apache/incubator-mxnet/pull/6283
Duplicated Open Issue: 
https://github.com/apache/incubator-mxnet/issues/7483
https://github.com/apache/incubator-mxnet/issues/6161

**Comment by xup6fup** [comment]
Do you want this file?

Lanking <notifications@github.com> 於 2018年4月5日 週四 02:05 寫道：

**Comment by anirudhacharya** [comment]
@lanking520 fixed in this PR - https://github.com/apache/incubator-mxnet/pull/10408

**Comment by lanking520** [comment]
@dslate Hi, we have added the old R documentation in this PR.
https://github.com/apache/incubator-mxnet/pull/10473
Hope this helps.

**Comment by dslate** [comment]
@lanking520   Thanks, I was able to download the new R mxnet manual at https://s3.amazonaws.com/mxnet-prod/docs/R/mxnet-r-reference-manual.pdf.  However, the page I originally referenced, http://mxnet.incubator.apache.org/api/r/index.html, still contains a link titled 'MXNet R Reference Manual' that points to the wrong document (mxnet-test.pdf).

**Comment by lanking520** [comment]
You are right. Once the PR merged, the link will be fixed.

**Comment by anirudh2290** [comment]
@lanking520 can we close this now ?

**Comment by lanking520** [comment]
Yes we can, the PR merged. The broken link should be fixed with next release

**Comment by lanking520** [comment]
@anirudh2290 Please close this Issue. We have updated the link on the website and we will add more support on R this summer.

**Comment by ankkhedia** [comment]
@sandeep-krishnamurthy Could you please close this issue as it has been resolved
